HIP/hip-514-missing-hts-precompile-functions.md

```diff
@@ -130,10 +130,16 @@ by the specific precompile function being called (see the functions below).
 
 ### Solidity Function Signatures
 
-Two functions are defined for creating fungible and non-fungible tokens each. One is a simple version with just the required fields and the other is a full version supporting custom fees. The ABI signature and hashes for each call are as follows:
+The ABI signature and hashes for token management functions are as follows:
 
 | hash      | effective signature                               | return                    |
 | ----------|---------------------------------------------------|---------------------------|
+| ``        | `allowance(address, address, address)`            | `(uint256)`               |
+| ``        | `approve(address, address, uint256)`              | `(uint256)`               |
+| ``        | `approveNft(address, address, uint256)`           | `(uint256)`               |
+| ``        | `isApproved(address, uint256)`                    | `(bool)`                  |
+| ``        | `isApprovedForAll(address, address, uint256)`     | `(bool)`                  |
+| ``        | `setApprovalForAll(address, address, bool)`       | `(bool)`                  |
```

Comment:
As it's a "write" I think we should stick with response codes.
```suggestion
| ``        | `setApprovalForAll(address, address, bool)`       | `(uint256)`                  |
```

Reply:
Updated